clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerUtils.java

```diff
@@ -134,44 +132,20 @@ public static FetchMetricsManager createFetchMetricsManager(Metrics metrics) {
     }
 
     public static <K, V> FetchConfig<K, V> createFetchConfig(ConsumerConfig config,
-                                                             Deserializer<K> keyDeserializer,
-                                                             Deserializer<V> valueDeserializer) {
-        IsolationLevel isolationLevel = createIsolationLevel(config);
-        return new FetchConfig<>(config, keyDeserializer, valueDeserializer, isolationLevel);
+                                                             Deserializers<K, V> deserializers) {
+        IsolationLevel isolationLevel = getConfiguredIsolationLevel(config);
+        return new FetchConfig<>(config, deserializers, isolationLevel);
     }
 
-    @SuppressWarnings("unchecked")
-    public static <K, V> List<ConsumerInterceptor<K, V>> createConsumerInterceptors(ConsumerConfig config) {
-        return ClientUtils.createConfiguredInterceptors(config,
-                ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG,
-                ConsumerInterceptor.class);
+    public static FetchConfig<String, String> createFetchConfig(ConsumerConfig config) {
+        Deserializers<String, String> deserializers = new Deserializers<>(new StringDeserializer(), new StringDeserializer());
```

Comment:
is this right?

Reply:
You're right, I don't think it is. This was defined like this in the commit we are cherry-picking here, that's why it ended up here, but I'll include the latest changes with the fix.

Reply:
would it make this PR clearer to remove this and add it to the FetchRequestManager PR?

Reply:
Sure, done. This was initially added as part of these changes though (and not as part of the FetchRequestManager PR), so we'll have to take care of that when cherry-picking the FetchRequestManager commit but that's fine.